### PR TITLE
11.0 mig product expiry simple

### PR DESCRIPTION
**Pull request by alan196**

### CODE REVIEW

**Comment by alan196** [comment]
Hello @alexis-via 

I just migrated the module product_expiry_simple to V11.
I also want to add the expiry date in the form when you create the lot, in order to set it when te lot is created.
Do you recommend this change in this module? or make a new module for that function?
What do you think about this?

Thanks in advance.

**Comment by pedrobaeza** [comment]
Check Travis and squash together "OCA Transbot..." commits with https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests

File: product_expiry_simple/__init__.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# © 2017 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
```
Comment (Nikul-Chaudhary) [inline product_expiry_simple/__init__.py]: @alan196  Remove Copyright from all `__init__.py`.

File: product_expiry_simple/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+{
+    'name': 'Product Expiry Simple',
+    'version': '11.0.1.0.0',
+    'category': 'Product',
+    'license': 'AGPL-3',
+    'summary':
+    'Simpler and better alternative to the official product_expiry module',
+    'author': 'Akretion,Jarsa Sistemas,Odoo Community Association (OCA)',
+    'website': 'http://www.akretion.com',
```
Comment (Nikul-Chaudhary) [inline product_expiry_simple/__manifest__.py]: @alan196 Replace URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10)

File: product_expiry_simple/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    'depends': [
+        'stock',
+    ],
+    'conflicts': [
+        'product_expiry'
+    ],
+    'data': [
+        'views/stock.xml',
+        'data/product_removal.xml',
```
Comment (Nikul-Chaudhary) [inline product_expiry_simple/__manifest__.py]: @alan196 Add `data` file first.

File: product_expiry_simple/models/stock_move_line.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+class StockMoveLine(models.Model):
+    _inherit = 'stock.move.line'
+
+    expiry_date = fields.Date(string='Expiry Date')
+
+    @api.model
+    def _action_done(self):
+        super(StockMoveLine, self)._action_done()
+        for rec in self:
+            if rec.move_id.picking_type_id.use_create_lots:
```
Comment (Nikul-Chaudhary) [inline product_expiry_simple/models/stock_move_line.py]: @alan196 here use 'and' condition.

Ex. `if rec.move_id.picking_type_id.use_create_lots and rec.lot_id:`

File: product_expiry_simple/views/stock.xml [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+  © 2017 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+<odoo>
+    <record model="ir.ui.view" id="view_stock_move_line_operations_lot_expiry_tree">
+        <field name="name">view.stock.move.line.operation.lot.expiry.tree</field>
+        <field name="model">stock.move.line</field>
+        <field name="inherit_id" ref="stock.view_stock_move_line_operation_tree"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='lot_name']" position="after">
```
Comment (Nikul-Chaudhary) [inline product_expiry_simple/views/stock.xml]: @alan196 IMO directly use `field`.

File: product_expiry_simple/views/stock_picking.xml [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+                <field name="expiry_date" groups="stock.group_production_lot" attrs="{'column_invisible': [('parent.show_lots_text', '=', False)], 'invisible': [('lots_visible', '=', False)], 'readonly': [('in_entire_package', '=', True)]}" context="{'default_product_id': product_id}"/>
+            </xpath>
+        </field>
+    </record>
+    <record id="view_picking_form_inherit" model="ir.ui.view">
+        <field name="name">product_expiry_simple.stock.move.operations.tree</field>
+        <field name="model">stock.move.line</field>
+        <field name="inherit_id" ref="stock.view_stock_move_line_operation_tree"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='lot_name']" position="after">
```
Comment (Nikul-Chaudhary) [inline product_expiry_simple/views/stock_picking.xml]: @alan196 same here

**Review by Nikul-Chaudhary — COMMENTED** [review]
Improve Code and change website URL

**Review by gurneyalex — APPROVED** [review]

**Comment by alexis-via** [comment]
@alan196 In v10, the module already adds the field expiry_date in the form view of stock.production.lot, so I don't understand your question.

**Review by alexis-via — APPROVED** [review]

**Pull request merged**